lib/setupmanagers/qemuhck/qemuhck.rb

```diff
@@ -59,7 +59,7 @@ def drivers_options
       options = {}
       @drivers&.each do |driver|
         OPT_NAMES.each do |name|
-          options[name] = driver[name] unless driver[name].nil?
+          options[name] = driver.instance_variable_get("@#{name}") unless driver.instance_variable_get("@#{name}").nil?
```

Comment:
Use `public_send`. It's not nice to read instance variables, which are inherently private, from outside.